src/webpack/index.ts

```diff
@@ -45,10 +49,10 @@ function buildManifest(compiler: Compiler, compilation: Compilation) {
 
         if (
           currentModule.rawRequest &&
-          !manifest[currentModule.rawRequest] &&
+          !manifest[name] &&
           !file.endsWith('.map')
         ) {
-          manifest[currentModule.rawRequest] = {
+          manifest[name] = {
```

Comment:
Do you know if `name` and `id` are now the same? Because that is the output I seem to get now

Reply:
Depends on webpack config. `id` is an identifier that likely becomes a number when you build with prod settings. While looks like `name` is the file path relative to the root folder.